modules/ROOT/nav.adoc

```diff
@@ -21,5 +21,15 @@
 ** xref:proc_interacting-with-pipelines-using-the-developer-perspective.adoc[4.2 Interacting with pipelines using the Developer perspective]
 * xref:assembly_cli-reference.adoc[5. CLI Reference]
 ** xref:proc_installing-cli.adoc[5.1 Installing CLI]
-** xref:ref_cli-reference.adoc[5.2 Basic Commands]
+** xref:op-tkn-cli-reference.adoc[5.2 Basic Commands]
+*** xref:op-tkn-cli-utility-commands.adoc[5.2.1 Utility Commands]
```

Comment:
In view of pushing these changes to OCP repo, and making them consistent with those docs, can you please change the topic names to:
op-tkn-cli-reference> tkn CLI Reference
- op-installing tkn> Installing tkn
- op-tkn-basic-commands> Basic tkn commands

Reply:
@Preeticp  - Thank you for your comments. I have incorporated the comments, and have also made changes to the headings of each module to be in sentence case like in OCP Docs  here - https://docs.openshift.com/container-platform/4.3/cli_reference/openshift_cli/administrator-cli-commands.html

Please review, and let me know if there are any more changes required.